### DISCUSSION TITLE
Home Assistant (MQTT): some data is almost always unknown

### DISCUSSION THREAD

**Discussion by spanzetta** [jomjol/AI-on-the-edge-device#2835] [General]
See below..
For most of the time I see unknown for all those data.. 

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/172eda3d-8c8a-42b4-a14e-90dddc58d6cf)

If I open their history I see that sometimes they contains the (right) data.. 

![image](https://github.com/jomjol/AI-on-the-edge-device/assets/49818743/14ebee21-b5b7-4acf-8a6d-0a4252354ec5)

Why it happens?

Thanks

**Comment by spanzetta** [comment]
Any help/suggestions?
Does it happen only to me?

**Comment by Slider0007** [comment]
Have you enabled the MQTT retain flag in device config?

**Comment by spanzetta** [comment]
With latest 15.5 the problem doesn't happen anymore..

**Reply by caco3** [reply]
Well, there is absolutely no change to the MQTT handling in that version!

Some static values like hostname, interval, IP only get sent on startup. Did you by any chance have Homeassistant (or your MQTT broker) switched off when you started the device laste time (before the restart with the 15.5)?
